safetypy/safetypy.py

```diff
@@ -492,6 +501,97 @@ def delete_response(self, responseset_id, response_id):
         log_message = 'on DELETE for response_set: {0}'.format(responseset_id)
         self.log_http_status(response.status_code, log_message)
 
+    def get_my_org(self):
+        """
+        GET organisation ID of the requesting user
+        :return: organisation ID of the user
+        """
+        response = self.authenticated_request_get('https://sandpit-api.safetyculture.io/share/connections')
```

Comment:
Sandpit? API url should be taken from the top level shared package

Reply:
Yes..this is what i mentioned yesterday, i have left them there so i can test it again after PR changes :) Will remove them before sending the final PR

Reply:
You can choose to not stage these changes and not commit them but only keep them on your local Git repo